updates/13xx_petguardians_update.sql

```diff
@@ -0,0 +1,33 @@
+-- These are but a fraction of the guardian pets, but most are with engi and need to be scripted. This will do for now.
+-- I am fixing the core to handle guardians by their values, check https://github.com/cmangos/mangos-classic/pull/116 for reference.
+
+-- Fixed Health of Timbermaw Ancestor (15720)
+UPDATE creature_template SET MinLevelHealth=500, MaxLevelHealth=500 WHERE Entry=15720;
```

Comment:
http://wowwiki.wikia.com/wiki/Defender_of_the_Timbermaw?oldid=289644
Didn't pull it from my rear end, that's for sure ;)